### PR TITLE
Add a rough guide for quick start

### PR DESCRIPTION
**Pull request by gobbq**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

Add a rough guide for RocketMQ deployment, include three different environments:
- Run RocketMQ locally
- Run RocketMQ in Docker
- Run RocketMQ in Kubernetes

## Brief changelog

add a rough guide for quick start

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by li-xiao-shuang** [comment]
This is a good idea, I think that would be better to put it on https://rocketmq.apache.org/docs/quick-start/

File: README.md [hunk 1]
```diff
@@ -34,6 +34,110 @@ It offers a variety of features:
 * Lightweight real-time computing
 ----------
 
+
+## Quick Start
+
+This paragraph guides you through steps of installing RocketMQ in different ways.
+For local development and testing, only one instance will be created for each component.
+
+### Run RocketMQ locally
```
Comment (caigy) [inline README.md:43]: Hope that quick start guides for Windows would be supplemented.

**Review by caigy — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -34,6 +34,110 @@ It offers a variety of features:
...
+
+Download the latest RocketMQ release from the Apache mirror:
+```shell
+$ wget https://archive.apache.org/dist/rocketmq/4.9.3/rocketmq-all-4.9.3-bin-release.zip
+```
+
+Unpack the release:
+```shell
+$ unzip rocketmq-all-4.9.3-bin-release.zip
+$ cd rocketmq-all-4.9.3-bin-release
```
[the remark this answers is not in the document]
Comment (gobbq) [reply README.md]: > You'd better not use static version (4.9.3) here.

That‘s what I think, but I didn't find a common method to express the release url of the latest version in Apache mirror.

**Comment by gobbq** [comment]
> This is a good idea, I think that would be better to put it on https://rocketmq.apache.org/docs/quick-start/

**Comment by gobbq** [comment]
> This is a good idea, I think that would be better to put it on https://rocketmq.apache.org/docs/quick-start/

Thanks, I think this is for that most people don't want to check out the official website.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (token01) [reply README.md]: ye

**Comment by xiaodaweic** [comment]
> > 这是个好主意，我认为将其放在[https://rocketmq.apache.org/docs/quick-start/上会更好](https://rocketmq.apache.org/docs/quick-start/)
> 
> 谢谢，我认为这是因为大多数人不想查看官方网站。

我认为大多数都会以官方网站做参考，至少我是这样的。现在rocket官方文档太简陋。我目前找不docker版本的rocket5。dockerhub上的现在是4.9.4
image

**Pull request merged**